control/latlon_muxer/CMakeLists.txt

```diff
@@ -1,33 +1,41 @@
-cmake_minimum_required(VERSION 3.0.2)
+cmake_minimum_required(VERSION 3.5)
 project(latlon_muxer)
 
-add_compile_options(-std=c++14)
+### Compile options
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic -Wno-unused-parameter)
+endif()
 
-find_package(
-  catkin REQUIRED COMPONENTS
-    roscpp
-    autoware_control_msgs
-)
+find_package(ament_cmake REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(autoware_control_msgs REQUIRED)
 
-catkin_package(
-  INCLUDE_DIRS include
-  CATKIN_DEPENDS
-    roscpp
-    autoware_control_msgs
-)
+add_executable(latlon_muxer src/node.cpp src/main.cpp)
 
-include_directories(
-  include
-  ${catkin_INCLUDE_DIRS}
+set(LATLON_MUXER_DEPENDENCIES
+  autoware_control_msgs
+  rclcpp
 )
 
-add_executable(latlon_muxer src/node.cpp src/main.cpp)
-target_link_libraries(latlon_muxer ${catkin_LIBRARIES})
-add_dependencies(latlon_muxer ${catkin_EXPORTED_TARGETS})
+ament_target_dependencies(latlon_muxer ${LATLON_MUXER_DEPENDENCIES})
+target_include_directories(latlon_muxer PRIVATE include)
+
+if(BUILD_TESTING)
+  find_package(ament_lint_auto REQUIRED)
```

Comment:
This looks good! I thought about including it but then decided to just port and not add extra features. BUt this linter testing is  a good topic for general code cleanup in the near future

Reply:
Yeah, I prefer ament_auto too, but I thought it was out of scope for the porting. I'm happy to switch to it.